app/views/early_allocations/new.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :switcher do %>
+  <%= render '/layouts/prison_switcher' %>
+<% end %>
+
+<% if @offender.errors.count > 0 %>
+  <%= render :partial => "/shared/validation_errors", :locals => { :errors => @offender.errors } %>
+<% end %>
+
+<h1>Assessment for early allocation to the community probation team</h1>
```

Comment:
h1's need a special class (otherwise it renders with Times as a font). There are a list of options at https://design-system.service.gov.uk/styles/typography/ ..

```
* class="govuk-heading-xl"
* class="govuk-heading-l"
* class="govuk-heading-m"
* class="govuk-heading-s">govuk-heading-s
```